network/wsNetwork_test.go

```diff
@@ -1923,6 +1927,13 @@ func TestWebsocketNetworkTXMessageOfInterestRelay(t *testing.T) {
 	waitReady(t, netB, readyTimeout.C)
 
 	netB.OnNetworkAdvance()
+	for {
+		// wait for async messagesOfInterestRefresh
+		time.Sleep(time.Millisecond)
+		if len(netB.messagesOfInterestRefresh) == 0 {
+			break
+		}
+	}
```

Comment:
nit: four copies of this, maybe worth a helper function. Maybe a generalized version of `waitReady`?
Not sure if it's a real concern here, but it's also an infinite loop if the condition is never true.